hack/local-up-cluster.sh

```diff
@@ -39,6 +39,8 @@ WAIT_FOR_URL_API_SERVER=${WAIT_FOR_URL_API_SERVER:-10}
 ENABLE_DAEMON=${ENABLE_DAEMON:-false}
 HOSTNAME_OVERRIDE=${HOSTNAME_OVERRIDE:-"127.0.0.1"}
 CLOUD_PROVIDER=${CLOUD_PROVIDER:-""}
+ETCD_HOST=${ETCD_HOST:-"127.0.0.1"}
```

Comment:
I wasn't aware of that `init.sh` script. People can call `local-up-cluster.sh` without calling `init.sh` (see http://kubernetes.io/docs/getting-started-guides/locally/#starting-the-cluster)